_data/toc/search-enterprise.yaml

```diff
@@ -10,4 +10,14 @@
     - title: 'Administer'
       path: '/search-enterprise/latest/admin/'
     - title: 'Using'
-      path: '/search-enterprise/latest/using/'	  
\ No newline at end of file
+      pages:
+        - title: 'Search query syntax'
+          path: '/search-enterprise/latest/using/'
+        - title: 'Filtering paging and sorting'
+          path: '/search-enterprise/latest/using/filtering'
+        - title: 'Field queries'
+          path: '/search-enterprise/latest/using/field'
+        - title: 'Query languages'
+          path: '/search-enterprise/latest/using/languages'
```

Comment:
Missing slash (/) at the end of URL. Therefore the left navigation will not show if selected.